Yury_Charnenkau/02.Git.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,32 @@
+## Git local
+
+git log --oneline #master:
+- a824277 (HEAD -> master) Merge branch 'hotfix/we_gonna_die'
+- 505020e (hotfix/we_gonna_die) hotfix/we_gonna_die commit
+- e7f1b4c features/do_one commit
+- 0466075 dev commit 2
+- 9826449 dev commit 1
+- 692c920 master commit 2
+- 1c59987 master commit 1
+
+git log --oneline #dev:
```

Comment:
please use code section here

ref: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code